Formula/g/gnupg.rb

```diff
@@ -39,19 +39,19 @@ class Gnupg < Formula
     depends_on "gettext"
   end
 
-  # Backport fix for missing unistd.h
-  patch do
-    url "https://git.gnupg.org/cgi-bin/gitweb.cgi?p=gnupg.git;a=patch;h=1d5cfa9b7fd22e1c46eeed5fa9fed2af6f81d34f"
-    sha256 "610d0c50004e900f1310f58255fbf559db641edf22abb86a6f0eb6c270959a5d"
-  end
-
   def install
     libusb = Formula["libusb"]
     ENV.append "CPPFLAGS", "-I#{libusb.opt_include}/libusb-#{libusb.version.major_minor}"
+    ENV.append "LDFLAGS", "-L#{libusb.opt_lib}"
+
+    readline = Formula["readline"]
+    ENV.append "CPPFLAGS", "-I#{readline.opt_include}"
+    ENV.append "LDFLAGS", "-L#{readline.opt_lib}"
```

Comment:
Please answer my question from the other review about notifying upstream about this

Reply:
There MUST be no notification about this.

`readline` is a keg-only formula, so it requires a special handling. It is shown when you install it like this:
```
$ brew uninstall --ignore-dependencies readline
Uninstalling /opt/homebrew/Cellar/readline/8.2.13... (51 files, 1.8MB)
<HOSTNAME>:~ <USERNAME>$ brew install readline
==> Auto-updating Homebrew...
Adjust how often this is run with HOMEBREW_AUTO_UPDATE_SECS or disable with
HOMEBREW_NO_AUTO_UPDATE. Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
==> Downloading https://ghcr.io/v2/homebrew/core/readline/manifests/8.2.13
######################################################################################################################################################### 100.0%
==> Fetching readline
==> Downloading https://ghcr.io/v2/homebrew/core/readline/blobs/sha256:738c27ceee9a8b198f98438477ef7a513a96a965e3a434ac3aa8fb4ed76494b1
######################################################################################################################################################### 100.0%
==> Pouring readline--8.2.13.arm64_sequoia.bottle.tar.gz
==> Caveats
readline is keg-only, which means it was not symlinked into /opt/homebrew,
because macOS provides BSD libedit.

For compilers to find readline you may need to set:
  export LDFLAGS="-L/opt/homebrew/opt/readline/lib"
  export CPPFLAGS="-I/opt/homebrew/opt/readline/include"

For pkg-config to find readline you may need to set:
  export PKG_CONFIG_PATH="/opt/homebrew/opt/readline/lib/pkgconfig"
==> Summary
🍺  /opt/homebrew/Cellar/readline/8.2.13: 51 files, 1.8MB
==> Running `brew cleanup readline`...
Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
```

Since it is Homebrew's decision to make it keg-only and that it's Homebrew's decision that those environment variables need to be set, it's not an issue for the upstream.
@SMillerDev has received his answer in https://github.com/Homebrew/homebrew-core/pull/200050#discussion_r1873589538
I believe, @SMillerDev is harassing me and is also incompetent.
@p-linnane @MikeMcQuaid This is a repeated offence!
A parrot has learned to say, "is it reported upstream?", and has become a Homebrew maintainer.
I hope you have higher standards. If not, I'd rather live in private taps.

As for `libusb`, I did add `LDFLAGS` for the sake of symmetry.
But the `CPPFLAGS` was added by a previous maintainer, because `libusb` installs headers in a non-standard location.
This issue is absolutely out of scope at the moment.